core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts

```diff
@@ -17,6 +17,7 @@ import { Jazz } from '../page-objects/jazzservices.po';
 import { CONFIGURATIONS } from '../../src/config/configuration';
 import { Timeouts } from 'selenium-webdriver';
 import { SSL_OP_SSLEAY_080_CLIENT_DH_BUG } from 'constants';
+import { async } from 'q';
```

Comment:
where is this being used? and see if you can use nodejs native async and await

Reply:
this is not required in the test suit by mistake it applied. removing this from the suit.